tests/generation/test_configuration_utils.py

```diff
@@ -18,9 +18,8 @@
 import tempfile
 import unittest
 import warnings
-from pathlib import Path
 
-from huggingface_hub import HfFolder, create_pull_request, create_repo, delete_repo
+from huggingface_hub import HfFolder, create_pull_request
```

Comment:
Not related to this PR but `huggingface_hub.HfFolder` is a deprecated class. Better to use `huggingface_hub.get_token` and `huggingface_hub.save_token` instead. Can be done in a later PR.